lib/languages/french.sty

```diff
@@ -0,0 +1,74 @@
+\ExplSyntaxOn
+% Use tilde (~) for spaces.
+
+% French captions
+\addto\captionsfrench
+  {
+    % Monster basics
+    \renewcommand\armorclassname{Classe~d'armure}
+    \renewcommand\hitpointsname{Hit~Points}
```

Comment:
Aren't these called "Points de vie" in the French material?

Reply:
Indeed, I forgot to translate this one.